#move_excavator/MultiAgent 

bool isRoverInView 
bool isRoverInRange 